apps/wallet/src/ui/app/pages/welcome/index.tsx

```diff
@@ -15,63 +16,67 @@ import PageLayout from '_pages/layout';
 
 import st from './Welcome.module.scss';
```

Comment:
Might be a good opportunity to remove this (if you got some time). Just replace the Link on line 84 with the [Button](https://github.com/MystenLabs/sui/blob/main/apps/wallet/src/ui/app/shared/ButtonUI.tsx) component and replace any other usages with tailwind.

Reply:
removed css module